packages/theme/components/BottomNavigation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+<!-- TODO: create logic with isActive prop for BottomNavigationItems -->
+  <SfBottomNavigation class="navigation-bottom smartphone-only">
+      <SfBottomNavigationItem
+        :class="$route.path == '/' ? 'sf-bottom-navigation__item--active' : ''"
+        icon="home" size="20px" label="Home"
+        @click="handleHomeClick"
+      />
+    <SfBottomNavigationItem icon="menu" size="20px" label="Menu" @click="toggleMobileMenu"/>
```

Comment:
Could you please add i18n to the labels here?

Reply:
done